doc/usecases/snippet_sharing/UC_edit_snippet.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Edit configuration snippet
+
+## Summary
+
+Title: Edit configuration snippet
+Scope: Entry Management
+Level: User Goal
+Actors: Authenticated user (snippet owner), Moderator
+Brief: Users can edit their own database entries. Moderators can edit all configuration snippets in the database, also the ones of other users.

```

Comment:
Thank you! Did you have time to investigate the github API? Can we use github for authentification (should be noted in the use cases, too)?

What might be great for Elektra development is some easy way to report issues that consideres context, e.g. if something is wrong with a plugin (README.md wrong, import fails,...) that a user already can create a new issue with the boring stuff already already filled out (Which plugin was used, what was the snippet that should be imported, what was the error message, and so on...).